doc/src/modules/holonomic/represent.rst

```diff
@@ -0,0 +1,40 @@
+Represnting a Holonomic Function in SymPy
```

Comment:
I would rather say something like: Representation of holomorphic functions in SymPy.